features/liquidity/hooks/usePoolTokensDollarValue.ts

```diff
@@ -1,14 +1,25 @@
 import { useSwapInfo } from 'hooks/useSwapInfo'
 import { useTokenDollarValue } from 'hooks/useTokenDollarValue'
-import { useBaseTokenInfo } from 'hooks/useTokenInfo'
+import {
+  PoolEntityType,
```

Comment:
what is PoolEntityType?

Reply:
Is a type for entities we have in pool json